pragma/tests/fetchers_test.py

```diff
@@ -46,12 +48,18 @@ async def test_async_fetcher(fetcher_config, mock_data):
             base_asset = asset["pair"][1]
 
             # FIXME: Adapt all fetchers and use `sync` decorator on fetchers
-            if fetcher_config['name'] == "AVNU":
-                mock.get(url, status=200, payload=mock_data[quote_asset])
+            if fetcher_config["name"] == "AVNU":
+                # AvnuFetcher._fetch_decimals is a coro that uses
+                # other coro's underneath which causes long delays for the test
+                # format_url_async -> _fetch_decimals -> call_contract -> call
+                mocker.patch.object(
+                    AvnuFetcher,
+                    "_fetch_decimals",
+                    return_value=18,
+                )
```

Comment:
Tbh, not sure if this is the best way since this is meant to be an integration test. Maybe a better method to mock would be `call_contract`.

Reply:
We could have the devnet node run in forked mode and use that yes, because here you are hardcoding 18 which is not the case for BTC it is 8.
Or we can mock the `call_contract`, wdyt is the most straightforward?

Reply:
I think running the devnet node is more straightforward!